### PR TITLE
Updated the payment page for Umbraco Cloud

### PR DESCRIPTION
**Pull request by jonat123**
- updated images
- updated the text

### CODE REVIEW

File: umbraco-cloud/set-up/manage-subscriptions/new-shop.md [hunk 1]
```diff
@@ -7,45 +7,49 @@ This article is only for customers in the payment system released on the 5th of
...
+Once a credit card has been added it will show up in a drop-down when creating new projects. You can also change the payment method for a specific project from here.
 
 ![Select Payment Methods](images/Payment_methods.png)
 
 ## Payment and Invoices
 
 On Umbraco Cloud, we are sending out one single invoice with all the projects that you are paying for via email every month.
 
-If you want to watch the Payment history and download an invoice from a previous month, you do this from the Payment setting in the dropdown menu on your projects.
+You can watch the invoices for your projects under your organization in the Payment history section. From here you can see the following for each invoice:
```
Comment (sofietoft) [inline umbraco-cloud/set-up/manage-subscriptions/new-shop.md:43]:
```suggestion
You can view the invoices for your projects under your organization in the Payment History section. From here you can see the following for each invoice:
```

File: umbraco-cloud/set-up/manage-subscriptions/new-shop.md [hunk 2]
```diff
@@ -7,45 +7,49 @@ This article is only for customers in the payment system released on the 5th of
...
 
-![Payment setting](images/payment-settings.png)
+* The payment number
+* The total amount paid
+* When the invoice was created 
+* The due date
+* The status of the invoice
+* Download the invoice
```
Comment (sofietoft) [inline umbraco-cloud/set-up/manage-subscriptions/new-shop.md:50]: This is not really something you can see, like with the other items on the list.

Perhaps this should be moved out of the list, and instead be mentioned in a sentence below. Possibly together with the final sentence in the article.

File: umbraco-cloud/set-up/manage-subscriptions/new-shop.md [hunk 3]
```diff
@@ -7,45 +7,49 @@ This article is only for customers in the payment system released on the 5th of
 For customers on Umbraco Cloud before the 5th of May, 2021 handling subscriptions is done through [the shop on Umbraco.com](https://shop.umbraco.com/profile/sign-in?returnURL=%2fprofile).
 {% endhint %}
 
-In this article, you will be able to find information on how to manage your subscriptions, download and pay invoices, and change your credit card on Umbraco Cloud.
+In this article, you will be able to find information on the following:
+- How to manage your subscriptions 
+- Download and pay invoices
+- And change your credit card on Umbraco Cloud.
```
Comment (sofietoft) [inline umbraco-cloud/set-up/manage-subscriptions/new-shop.md:13]:
```suggestion

- How to manage your subscriptions.
- How to download and pay invoices.
- How to change your credit card for payments.
```

**Review by sofietoft — COMMENTED** [review]
Looks great. A few suggestions - mainly around some formatting 💪

File: umbraco-cloud/set-up/manage-subscriptions/new-shop.md (hunk 2 above)
Comment (jonat123) [reply umbraco-cloud/set-up/manage-subscriptions/new-shop.md]: @sofietoft  Makes sense to me, I've moved it out and added it under the image.

**Comment by jonat123** [comment]
Merging in the PR.

**Pull request merged**